herculens/Util/jax_util.py

```diff
@@ -26,6 +33,43 @@ def unjaxify_kwargs(kwargs_params):
     return kwargs_params_new
 
 
+def R_omega(z, t, q, nmax):
```

Comment:
just a comment: This probably resembles some functions that were implemented here: https://github.com/sibirrer/lenstronomy/blob/main/lenstronomy/LensModel/Profiles/epl_numba.py

Reply:
Oh, in fact we did not about this one... if I am correct, @austinpeel had to recode this function from scratch, as he was not aware of this implementation (but I let him confirm that).